dev/tasks/python-wheels/win-build.bat

```diff
@@ -22,11 +22,19 @@
 
 @rem --file=arrow\ci\conda_env_gandiva.yml ^
 
-@rem create conda environment for compiling
 @rem FIXME: Update to numpy 1.16.6
+SET NUMPY_VERSION=1.16.5
+SET VS_VERSION=
+if %PYTHON_VERSION% == 3.6 (
+    SET VS_VERSION="<14.16"
```

Comment:
```suggestion
    SET VS_VERSION=<14.16
```

Reply:
That's what I had here https://github.com/ursa-labs/crossbow/runs/1672510076 and it looked like it was treating it as interpolation. I'm not sure if it's not liking the multiline or the `else if`.

Reply:
I'm glad I don't need to deal with batch scripting on a regular basis, but it seems like this should have been `SET VS_VERSION=^<14.16` based on https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/set_1#remarks